urdf/ergoCub/robots/ergoCubGazeboV1_minContacts/model.urdf

```diff
@@ -3291,6 +3298,20 @@
     <parent link="head"/>
     <origin rpy="3.141592653589793 -0.0 1.5707963267948966" xyz="-0.004299998 -0.0055 0.03823499999999996"/>
   </sensor>
+  <gazebo reference="torso_1">
+    <sensor name="waist_imu_0" type="imu">
+      <always_on>1</always_on>
+      <update_rate>100</update_rate>
+      <pose>0.0354497 0.00639688 0.060600 -1.5707322 1.5707963 0</pose>
+      <plugin name="ergocub_yarp_gazebo_plugin_IMU" filename="libgazebo_yarp_imu.so">
+    <yarpConfigurationFile>model://ergoCub/conf/gazebo_ergocub_inertial.ini</yarpConfigurationFile>
```

Comment:
.ini file should be gazebo_ergocub_waist_inertial.ini

Reply:
Done here https://github.com/icub-tech-iit/ergocub-software/pull/99/commits/72a2fda9265cfaf6f5ea467af2cb0dac03d755eb